src/components/shared/Project/ProjectHeader/index.tsx

```diff
@@ -98,23 +103,34 @@ export default function ProjectHeader({
                   fontSize: '0.8rem',
                   color: colors.text.disabled,
                   textTransform: 'uppercase',
-                  marginRight: spacing,
                 }}
               >
                 (archived)
               </span>
             )}
-            {handle && (
+            {handle ? (
               <span
                 style={{
                   color: colors.text.secondary,
-                  marginRight: spacing,
                   fontWeight: 600,
                 }}
               >
                 @{handle}
               </span>
-            )}
+            ) : onClickSetHandle ? (
```

Comment:
if no handle is set and owner wallet is connected, show button prompt to set handle